modules/installation-installing-bare-metal.adoc

```diff
@@ -67,6 +67,24 @@ INFO It is now safe to remove the bootstrap resources
 The command succeeds when the Kubernetes API server signals that it has been
 bootstrapped on the control plane machines.
 
+. Check that all the configured bare-metal hosts are available in the cluster:
++
+----
+oc get bmh -A
+----
++
+.Example output
+[source,terminal]
+NAMESPACE               NAME               STATE         CONSUMER                      ONLINE 
+openshift-machine-api   example-compute1   inspecting                                   true   
+openshift-machine-api   example-compute2   provisioned   example-tlms8-worker-0-j9x86   true
+openshift-machine-api   example-compute3   inspecting                                   true
+openshift-machine-api   example-compute4   provisioned   example-tlms8-worker-0-k2qkk   true
+openshift-machine-api   example-control1   provisioned   example-tlms8-master-0         true
+openshift-machine-api   example-control2   provisioned   example-tlms8-master-1         true
+openshift-machine-api   example-control3   provisioned   example-tlms8-master-2         true
+
+
```

Comment:
Maybe we should use here a "successful" example (that is, all bmh _provisioned_).

Reply:
Replaced with all "provisioned" nodes.

Reply:
Hi @rohennes , please note that apart of including `worker` nodes in the example here, which makes no sense for bootstrap process, this documentation is for UPI BM, AFAIK, `oc get bmh` is a command that will not return any resource in this case.